### PR TITLE
Add method gameWon()

### PR DESCRIPTION
**Pull request by julia-kim**
#4 Added a method to check if game is won. I used Jared's idea of mapping Os and Xs to 1s and -1s. Have not addressed whether game is in an end state + results.

### CODE REVIEW

File: src/com/nick/ttt/TicTacToeGame.java [hunk 1]
```diff
@@ -47,20 +69,45 @@ public void drawBoard() { // visual display for the human
     }
 
     public void placePiece(char xo, int pos) {
-        this.getGameBoard()[pos] = xo;
+        if (this.findEmpties().contains(pos)) {
```
Comment (nstoltzfus3) [inline src/com/nick/ttt/TicTacToeGame.java]: The check that verifies a valid move should be separate from the method that places it. Place Piece can make a call to the verifier. But the verifier should be distinct (we will use it for our AI.)

File: src/com/nick/ttt/TicTacToeGame.java [hunk 2]
```diff
@@ -47,20 +69,45 @@ public void drawBoard() { // visual display for the human
...
         for (int i = 0; i < board.length; i++) {
-            if (board[i] == 0) {
+            if (board[i] == ' ') {
                 empties.add(i);
             }
         }
         return empties;
     }
 
+    public boolean gameWon() {
```
Comment (nstoltzfus3) [inline src/com/nick/ttt/TicTacToeGame.java:86]: The intuition is correct here! Looks good.

**Review by nstoltzfus3 — COMMENTED** [review]
Pull out the verifier and it looks good!

**Comment by nstoltzfus3** [comment]
I'll merge this as soon as the small change is made.

**Comment by julia-kim** [comment]
Done!

**Comment by nstoltzfus3** [comment]
Awesome!

**Pull request merged**